salt/modules/zypper.py

```diff
@@ -115,6 +115,11 @@ def info_installed(*names):
         t_nfo = dict()
         # Translate dpkg-specific keys to a common structure
         for key, value in pkg_nfo.items():
+            try:
+                value = value.encode('UTF-8', errors='replace')
```

Comment:
I thought about this too but hoped that `errors='replace'` would keep more bad strings (kind of) readable.